.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: ruby
+cache: bundler
+
+before_script:
+- cp botconfig.yml.example botconfig.yml
```

Comment:
`botconfig.yml.example` probably has bad data in it. If we don't care about it actually running, it'd probably be fine.